app/models/report.rb

```diff
@@ -1,8 +1,8 @@
 class Report < ActiveRecord::Base
   validates :user_id, presence: true
   validates :item_id, presence: true
-  validates :item_type, presence: true, :inclusion => { :in => %w(post comment),
-    :message => 'Type should match `post` or `comment`!'}
+  validates :item_type, presence: true, :inclusion => { :in => %w(Post Comment),
+    :message => 'Type should match `Post` or `Comment`!'}
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.